docs/storage-abstraction.md

```diff
@@ -47,18 +69,27 @@ You can create a JSON file emoney-storage.json with the JSON below.
 }
 ```
 
-Note that the `transaction` field is set to `false`, which indicates you use this table with the Storage API.
+To apply the schema, go to the [ScalarDB Releases](https://github.com/scalar-labs/scalardb/releases) page and download the ScalarDB Schema Loader that matches the version of ScalarDB that you are using to the `getting-started` folder.
 
-To apply the schema, download the Schema Loader that matches the version you use from [scalardb releases](https://github.com/scalar-labs/scalardb/releases), and run the following command to load the schema.
+Then, run the following command, replacing `<VERSION>` with the version of the ScalarDB Schema Loader that you downloaded:
 
-```
-$ java -jar scalardb-schema-loader-<version>.jar --config scalardb.properties -f emoney-storage.json
+```console
+$ java -jar scalardb-schema-loader-<VERSION>.jar --config scalardb.properties -f emoney-storage.json
 ```
 
 ### Example code
 
-This is a simple electronic money application with the Storage API.
-(Be careful: it is simplified for ease of reading and far from practical and is certainly not production-ready.)
+The following is the source code for the electronic money application that uses the Storage API.
+
+To use this source code, create a file named `ElectronicMoneyStorage.java` in the `scalardb/docs/getting-started/src/main/java/sample` directory. Then, add the following Java code to the file.
```

Comment:
@brfrn169 After re-adding the Java source code to this doc, I added these sentences since I felt like we should guide users on what to do with the source code. What do you think about this part?

Reply:
@josh-wong I'm not sure just copying the source code works. Did you test is and check if it works?

Reply:
@brfrn169 Sorry, I made an assumption here since I didn't see instructions on what users should do with this source code.

When I try the method I'm seeing errors when I try running this code in the way that I described. I've also tried copying this code to `ElectronicMoney.java` and pasting it over the existing code, but I still encounter errors.

Could you please tell me where users should apply this source code to so that we can provide some guidance?

Reply:
@josh-wong How about we just place the example source code here and skip the detailed run instructions because they might get a bit tricky? Given this document is for experts, I feel it should be fine. What do you think?

Reply:
@brfrn169 I'm OK with that for now. I've removed the instructions that I previously added and changed this part to be informational as it was before. We can update this section later based on feedback, if necessary.